vortex-array/src/compute/list.rs

```diff
@@ -68,13 +68,14 @@ pub fn list_contains(array: &dyn Array, value: &dyn Array) -> VortexResult<Array
     if let Some(value_scalar) = value.as_constant() {
         list_contains_scalar(array, &value_scalar, nullability)
     } else if let Some(list_scalar) = array.as_constant() {
-        list_scalar_contains_array(&list_scalar.as_list(), value, nullability)
+        list_scalar_contains_constant_array(&list_scalar.as_list(), value, nullability)
     } else {
         todo!("unsupported list contains with list and element as arrays")
     }
 }
 
-fn list_scalar_contains_array(
+// Then there is a constant list scalar (haystack) being compared to an array of needles.
+fn list_scalar_contains_constant_array(
```

Comment:
to me this still reads like "check if the list scalar contains a constanarray as a subarray" or something, but i'm not gonna block on this

Reply:
Wait, this is not what it means?

Reply:
happy to rename to anythingg